miRge.pl

```diff
@@ -36,7 +36,7 @@
 my $version = "2.0";
 local $ENV{PATH} = "$ENV{PATH}:".$miRgePath;
 $miRgePath =~ s/\/[^\/]+\.pl/\//;
-my $refPath = $miRgePath."miRge.seqLibs/";
+my $refPath = $miRgePath."usr/local/miRge/miRge.seqLibs/";
```

Comment:
hard coding /usr/local/miRge into the miRge path is not something that we would want to do as it would make the code more inflexible to various installation setups